packages/type-definitions/src/index.ts

```diff
@@ -113,27 +113,15 @@ const defaultTypesBundle: OverrideVersionedType[] = [
 // Current runtime version: 10730
 export const typesBundle: OverrideBundleType = {
   chain: {
-    'KILT Spiritnet': {
+    'kilt-spiritnet': {
```

Comment:
Are you sure this works? `chain` refers to the chain name, while `spec` refers to the spec name. Here I wanted to use the chain name because `mashnet-node` spec includes both standalone and `peregrine` (which is not ideal),